src/client/pages/OfferNew/utils.ts

```diff
@@ -234,6 +234,12 @@ export const maskAndFormatRawSsn = (ssn: string) => {
   return ssn.substr(CENTURY_LENGTH, DATE_LENGTH) + '-****'
 }
 
+export const ssnLengthByMarket: Record<Market, number> = {
+  SE: 12,
+  NO: 11,
+  DK: 10,
+}
```

Comment:
Maybe we could consider handling both 12 and 14 characters at some point. The backend can handle both and if the user wants to input it with 19 at the start, why not let them 🤷

Reply:
You mean both 10 and 12 characters? 🙊

Reply:
Yup, that's allowed now @vonElfvin as you pointed out @styrelius  :)